src/test/resources/application-integration-test.properties

```diff
@@ -0,0 +1,21 @@
+server.port=8080
+
+db.janitor.uri=jdbc:postgresql://127.0.0.1:5432/testdb
+db.janitor.username=dbuser
+db.janitor.password=dbpwd
+db.janitor.recreateDbOnStart=true
+db.janitor.updateDbOnStart=true
+db.stairway.uri=jdbc:postgresql://127.0.0.1:5432/testdb_stairway
+db.stairway.username=dbuser_stairway
+db.stairway.password=dbpwd_stairway
+# Testing is easier if we are not scheduling changes to the database by default.
+janitor.primary.schedulingEnabled=false
+stairway.maxParallelFlights=40
+stairway.migrateUpgrade=true
+stairway.forceCleanStart=true
+stairway.quietDownTimeout=20s
+stairway.terminateTimeout=5s
+pubsub.track-resource.pubsubEnabled=true
+pubsub.track-resource.projectId=terra-kernel-k8s
+pubsub.track-resource.subscription=crljanitor-toolsalpha-pubsub-sub
+test.trackResourceTopicId=crljanitor-toolsalpha-pubsub-topic
```

Comment:
where does this value come from? Does it need to be this value?

Reply:
those values come from vault. It need to be some values which points to read Google cloud resource.

Reply:
Sorry, I don't really understand your reply. If these are supposed to come from vault, why are they hard coded here instead of rendered as config? As-is, these are magic values without comment.

Reply:
Maybe add a link to the terraform module or deployment to where these are originating from. then LG.
Let's file a placeholder bug to TODO here.